### ISSUE TITLE
A space is inserted after slash-star /* comments

### ISSUE THREAD

**Issue by cgrushko** [facebook/ktfmt#185]
ktfmt emits the following:
```
f(43
    /*bla */ )
```

(when the max line width is short enough)

Note the space between the closing `*/` and the closing `)`.

This is coming from https://github.com/google/google-java-format/blob/506875dc3f0f1018676c764e5fe58c9a8c216900/core/src/main/java/com/google/googlejavaformat/OpsBuilder.java#L510

Upstream bug: https://github.com/google/google-java-format/issues/569

**Comment by bethcutler** [comment]
Adding a few more examples.

Same as above, except it's more obvious that the spacing before and after the comment is inconsistent:
```kotlin
    doStuff(/* inline comment */ )
```

The same thing happens with curly braces, but in the opposite way:
```kotlin
  fun doSomething() {
    something { /* comment */}
  }
```

In both of these cases, ktlint flags the spacing after the comment (unexpected or missing) as an error, on different rules: "paren-spacing" on the first "curly-spacing" for the second.

**Comment by hick209** [comment]
I've just tested. This issue is still current

**Comment by hick209** [comment]
Fixed on current version, 0.49

Closing issue
